hack/boskos.py

```diff
@@ -37,7 +41,15 @@ def checkout_account(resource_type, user, input_state="free"):
         print(f"export BOSKOS_RESOURCE_NAME={result['name']}")
         print(f"export AWS_ACCESS_KEY_ID={result['userdata']['access-key-id']}")
         print(f"export AWS_SECRET_ACCESS_KEY={result['userdata']['secret-access-key']}")
-
+    # The http API has two possible meanings of 404s - the named resource type doesn't exist or there no available resources of the type.
+    # For our purposes, we don't need to differentiate.
+    elif r.status_code == 404:
+        print(f"could not find available host, retrying in {RETRY_WAIT}s")
+        if retries == MAX_RETRIES:
+            raise Exception(f"could not allocate host after {MAX_RETRIES} tries")
+        tries = tries + 1
```

Comment:
I think there is a tries vs retries naming issue here

Reply:
Doh, thanks!